### ISSUE TITLE
Add request upload limits to all REST APIs

### ISSUE THREAD

**Issue by malinthaprasan** [wso2/product-apim#10886]
### Description:
Need to add following configs to all REST APIs' beans.xml

Admin REST API is already configured.

```
    <jaxrs:server id="services" address="/">
        <jaxrs:serviceBeans>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.PoliciesApi"/>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.ThrottlingApi"/>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.ApplicationsApi"/>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.MonetizationApi"/>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.WorkflowsApi"/>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.TenantInfoApi"/>
            <bean class="org.wso2.carbon.apimgt.rest.api.admin.v1.ApiCategoriesApi"/>
....
        <jaxrs:properties>
            <!-- This is added to catch interceptor level exceptions in GlobalThrowableMapper. -->
            <entry key="map.cxf.interceptor.fault" value="true" />
            <!-- This is added to restrict the size of attachments sent through the requests. -->
            <entry key="attachment-max-size" value="10485760"/>
        </jaxrs:properties>
    </jaxrs:server>
```
Above will limit to 10MB max

### Steps to reproduce:
1. Create an API
2. Attach a document with about 20MB size. 
3. Upload get successful.

**Comment by HiranyaKavishani** [comment]
Reopen the issue to backport the fix for APIM 3.2.0

**Comment by HiranyaKavishani** [comment]
Close the issue as the fix has been shipped
